modules/maimai/libraries/maimaidx_apidata.py

```
@@ -21,8 +21,7 @@ async def update_cover() -> bool:
     id_list = ['00000', '01000']
     for song in (await total_list.get()):
         id_list.append(song['id'])
-    if not os.path.exists(mai_cover_path):
-        os.makedirs(mai_cover_path)
+    os.makedirs(mai_cover_path, exist_ok=True)
```

Ruff (0.8.0)

`os` may be undefined, or defined from star imports

(F405)